src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/fehandledoffline/FeHandledOfflineHandler.java

```diff
@@ -17,7 +16,7 @@
 public class FeHandledOfflineHandler implements PreSubmitCallbackHandler<SscsCaseData> {
     @Override
     public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
-        return callback != null && EventType.FURTHER_EVIDENCE_HANDLED_OFFLINE == callback.getEvent()
+        return callback != null
```

Comment:
Doesn't this need the event type? Otherwise it could trigger for all types of event